src/app/app.module.ts

```diff
@@ -123,6 +124,7 @@ import { TranslatableMatPaginator } from "./core/translation/TranslatableMatPagi
     ConfigurableEnumModule,
     ReportingModule,
     EntityUtilsModule,
+    MatProgressBarModule,
```

Comment:
This should be moved to the `AttendanceModule` To keep the top-level `AppModule` a little cleaner.